Sources/BraintreeCore/Analytics/BackgroundTaskManaging.swift

```diff
@@ -10,4 +10,13 @@ extension UIApplication: BackgroundTaskManaging {
     func beginBackgroundTask(named: String?, expirationHandler handler: (() -> Void)?) -> UIBackgroundTaskIdentifier {
         beginBackgroundTask(withName: named, expirationHandler: handler)
     }
+    
+    var applicationStateString: String {
```

Comment:
We should move this into a new file. It doesn't appear to related to sending analytics as a background task. We can name the new file in Core `UIApplication+ApplicationState`